configurator/DEBIAN/control.template

```diff
@@ -0,0 +1,12 @@
+Source: pantos-service-node-configurator
+Section: contrib/python
+Priority: extra
+Maintainer: Pantos GmbH <contact@pantos.io>
+Standards-Version: 3.9.5
+Homepage: https://github.com/pantos-io/servicenode
+Package: pantos-service-node-configurator
```

Comment:
The package name seems confusing to me since the package is not only configuring but also installing the Service Node (due to its dependencies). I would prefer a name like `pantos-service-node-full`.